Project 1 Day 2/application.py

```diff
@@ -66,4 +70,16 @@ def register():
         sys.stdout.flush()
         return 'Registration Failed'
     return 'Registration Success'
+ 
+#TASK - 6 DISPLAY RECORDS FOR ADMIN USING ADMIN.html page 
+@app.route("/admin")
+def admin():
+    try:
+        users = Dataentry.query.all()
+
+    except Exception as e:
+        print(e)
+        sys.stdout.flush()
+        return 'Admin Failed'
```

Comment:
This is okay for now, you may want to think of giving a more meaningful error message based on the type of exception raised.